awsshell/wizard.py

```diff
@@ -71,6 +71,10 @@ def __init__(self, session=None, interaction_loader=None,
         self._interaction_loader = interaction_loader
         if interaction_loader is None:
             self._interaction_loader = InteractionLoader()
+        if delegation_loader is None:
```

Comment:
I made the `delegation_loader` an init parameter to facilitate unit testing. Initially, the loader that was passed was always `self` which meant that if a wizard would delegate it would always attempt to load the wizard from disk. This allows me to pass a stub/mock for the delegation loader. Using `self` as the `delegation_loader` is intended to be the default behavior. I can improve the docstring for this.